pshchegolevatykh/ReverseList.cs

```diff
@@ -0,0 +1,37 @@
+/**
+ * Definition for singly-linked list.
+ * public class ListNode {
+ *     public int val;
+ *     public ListNode next;
+ *     public ListNode(int val=0, ListNode next=null) {
+ *         this.val = val;
+ *         this.next = next;
+ *     }
+ * }
+ */
+public class Solution {

```

Comment:
Cool! New clever way of solving for me, that we can actually build a reversed list alongside the existing one.
Somehow I always understood such task as "reverse the exact given linked list" 🙂  Nice way to go! 👍

Reply:
interesting solution :)